test/rails_apps/authlogic/config/environments/test.rb

```diff
@@ -14,7 +14,7 @@
   # Eager loading loads your whole application. When running a single test locally,
   # this probably isn't necessary. It's a good idea to do in a continuous integration
   # system, or in some way before deploying your code.
-  config.eager_load = ENV["CI"].present?
+  config.eager_load = false
```

Comment:
I spent a bunch of time trying to figure out why this was happening and wasn't able to get to the bottom of it. Just disabling eager loading for authlogic seemed to be the easiest workaround. Just for background, enabling `eager_load` causes problems with authlogic itself loading properly.